### DISCUSSION TITLE
Is there a way that i could move the h2 head to the right?

### DISCUSSION THREAD

**Discussion by yzqdev** [vuepress/core#737] [Q&A] [answered]
The default h2 head in markdown shows on the left, so is there any config that i could move the headers to the right? Or i should write some components,js or styles? i searched the doc but found no solution. 

![image](https://user-images.githubusercontent.com/20037151/156698067-25f893d3-0591-459f-ab74-eda930621280.png)

as is shown below, the h1 header is on the left and h2 header is on the right.

![image](https://user-images.githubusercontent.com/20037151/156698253-be77b625-cb71-4a82-8fbb-2af2322fe897.png)

**Comment by Mister-Hope** [comment, marked as answer]
You can use 3rd party theme to achieve this feature, we would like to keep the default theme simple, and let users make full use with our plugin api and theme api.

I would like to give a recommandation to my theme here, take a look at : https://vuepress-theme-hope.github.io/v2/

**Reply by yzqdev** [reply]
Awesome theme, thank you!:smiley:
